tests/Sdk/Unit/Infrastructure/Logger/TransactionNameNewRelicProcessorTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Sdk\Unit\Infrastructure\Logger;
+
+use Codeception\Test\Unit;
+use SprykerSdk\Sdk\Infrastructure\Logger\TransactionNameNewRelicProcessor;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group Unit
+ * @group Infrastructure
+ * @group Logger
+ * @group TransactionNameNewRelicProcessorTest
+ * Add your own group annotations below this line
+ */
```

Comment:
It's doesn't look like we need these annotations

Reply:
Let it be. I have an idea.